Dockerfile

```diff
@@ -1,31 +1,49 @@
 FROM node:12-buster
 LABEL maintainer="Kitware, Inc. <kitware@kitware.com>"
 
-EXPOSE 8080
-
-RUN mkdir /girder
-
 RUN apt-get update && apt-get install -qy \
     gcc \
     libpython3-dev \
     git \
     libldap2-dev \
-    libsasl2-dev && \
-  apt-get clean && rm -rf /var/lib/apt/lists/*
-
-RUN wget https://bootstrap.pypa.io/get-pip.py && python3 get-pip.py
-
-WORKDIR /girder
-COPY . /girder/
+    libsasl2-dev \
+    python3-pip \
+&& apt-get clean && rm -rf /var/lib/apt/lists/* \
+&& python3 -m pip install --upgrade \
+    pip \
+    setuptools \
+    setuptools_scm \
+    wheel
 
 # See http://click.pocoo.org/5/python3/#python-3-surrogate-handling for more detail on
 # why this is necessary.
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-# TODO: Do we want to create editable installs of plugins as well?  We
-# will need a plugin only requirements file for this.
-RUN pip install --upgrade --upgrade-strategy eager --editable .
+RUN mkdir /girder
+WORKDIR /girder
+COPY . /girder/
+
+# Build girder wheel file, and install it
+RUN python3 setup.py bdist_wheel \
+ && cd dist && python3 -m pip install girder && cd .. \
+ && rm -rf build dist
+
+# Build and install all plugins found in `plugins` directory
+RUN cd plugins \
+ && for plugin in `ls -d *`;\
+    do \
+    echo Building ${plugin} \
+    && cd ${plugin} \
+    && python3 setup.py bdist_wheel \
+    && cd dist && python3 -m pip install girder-`echo ${plugin} | tr _ -` && cd .. \
+    && rm -rf build dist \
+    && cd ..;\
+    done \
+ && cd ..
```

Comment:
This part (installing all plugins) seems unrelated to the bugfix, and could potentially require more discussion as to whether it's appropriate. Could we make that as a separate PR so we can get the bugfix part merged easily?